code/game/dna/dna_modifier.dm

```diff
@@ -330,12 +335,11 @@
 	var/injector_ready = FALSE	//Quick fix for issue 286 (screwdriver the screen twice to restore injector)	-Pete
 	var/obj/machinery/dna_scannernew/connected = null
 	var/obj/item/disk/data/disk = null
-	var/selected_menu_key = null
+	var/selected_menu_key = "ui"
```

Comment:
Feel like this should be a define, as well as the other menu keys

Reply:
Agreed